proto/projects/project.proto

```diff
@@ -2,12 +2,11 @@ syntax = "proto3";
 package lavanet.lava.projects;
 
 import "gogoproto/gogo.proto";
-
 option go_package = "github.com/lavanet/lava/x/projects/types";
 
 message Project {
     string index = 1; // unique id that will be the combination of subscription address and project name, cannot be changed once created
-    string subscription = 2; // the subscription address that owns the project
+    string subscription = 2; // the subscription address that owns the project (not necessarily the address that paid for the subscription. It's the address the subscription was bought for)
```

Comment:
why is this comment needed? "owns" is precise and concise.

Reply:
done